docs/examples/tips_and_tricks.rst

```diff
@@ -10,14 +10,20 @@ Tips & Tricks
 Including sunk costs for previous investments into specific assets
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-It is possible to include the previous investment costs into now pre-existing asset capacities in the economic evaluation of a scenario. For this, following trick can be applied to the input data of the asset in question, by setting:
+Usually, the investments into existing capacities are neglected and assumed to be sunk costs of the system. The existing capacity :code:`installedCap` as well as the age of the installed asset :code:`age_installed` are only used to calculate when necessary re-investments take place, and how high the :ref:`replacements costs <Cost_calculations>` are.
+
+When optimizing a system with pre-existing capacities of certain assets, it can be usefull for the user to implement the installation costs of these assets in the economic evaluation.
+
+With this, it is possible to optimize a system with a specific or a specific minimal capacity of a certain asset and still account for installation costs of the asset at the beginning of the project (in the idea of a greenfield / brownfield optimization). This can work for energy production assets as well as energy conversion assets.
```

Comment:
What does `this` refers to in that case (the question is valid for both `With this` and `This can work`? This sentence can be better formulated. `in the idea of a greenfield / brownfield optimization` is rather vague, what do you want to say?